web/src/main/java/org/hillview/targets/TableTarget.java

```diff
@@ -104,19 +104,28 @@ public void prune(RpcRequest request, RpcRequestContext context) {
         this.runPrune(this.table, new EmptyTableMap(), (d, c) -> new TableTarget(d, c, this.metadataDirectory), request, context);
     }
 
+    static class BasicColStatsArgs {
+        String[] cols;
+        long[] seeds;
+    }
+
     @HillviewRpc
     public void basicColStats(RpcRequest request, RpcRequestContext context) {
-        String[] args = request.parseArgs(String[].class);
-        BasicColStatSketch sk = new BasicColStatSketch(args, 2);
-        PostProcessedSketch<ITable, JsonList<BasicColStats>, JsonList<BasicColStats>> post =
+        BasicColStatsArgs args = request.parseArgs(BasicColStatsArgs.class);
+        BasicColStatSketch sk = new BasicColStatSketch(args.cols, 2, args.seeds);
+        PostProcessedSketch<ITable, JsonList<Pair<BasicColStats, HLogLog>>, JsonList<Pair<BasicColStats, CountWithConfidence>>> post =
```

Comment:
Is it really Count With Confidence?

Reply:
`HLogLog.getCount()" returns a `CountWithEvidence` object, but the `confidence` field is never updated after initializing to 0. How would you like me to resolve this? Add some code to compute a confidence interval, or just remove the `confidence` field?

Reply:
@uwieder is there a formula for the confidence interval of distinct count using hyperloglog?